### PR TITLE
BS Teleporter, Disk, & Loadout Changes + Gun Condensing

### PR DESCRIPTION
**Pull request by Stutternov**
## About The Pull Request
**Tl;dr -** removed some guns that were bloat-y and followed the vast majority of the vote held in Emojitocracy regarding Blackshield's prefered firearm choices. As such I've condensed down a fair bit of guns in standard rotation as well as 'faction unique' guns that had debatable purposes. I.e the Marshal Viper or the BS Watchtower.

Votes went as followed:
- Vote#2; PPSH won the vote. PPSH colony-version has replaced the Buckler.
- Vote#3; Saiga shotgun won; as such it has been added to the Sergeant loadout. Bounty made standard shotgun since fits BS better that Grizzly, Rushing Bull renamed the 'Buffalo' and moved to CO unique weapon as-per BS channel player suggestion.
- Vote#4; STS-25 carbine won - however upon discussion is removed (left in code but removed fully from rotation), the Duty has been **upgraded** to fit the STS-25's role better and has been buffed as per player feedback. Solarian slotted for future evaluation.
- Vote#5; Vintorez DMR won the vote. Watchtower DMR has been removed, Strelki slotted for future evaluation and more-than-likely removal. Kept currently for Lodge.
- Vote#6; STS-35/PARA won the vote. (I hate you all.... I regret spriting that gun so much.......)
- Vote#7; Mosin won the vote. Roe upon player request re-added to the tele and loadouts due to scope. Mosin kit has lost its round-start scope but may take scopes still. Zatvor slotted for future evaluation due to player request.
- Vote#8; Heroic LMG won. Maxim and Pegasus removed from disk/teleporter. Heroic kept.
- Vote on the Disk; 'Move high-cost weapons to disk, keep low-cost on tele' won. Moved all .408, LMG, or 'advanced weaponry' over to the disk. Kept standard guns on the Tele.

Non-vote changes:
- Drozd moved to being a 10mm SMG; mostly as a buff to Excelsior antags.
- Marshal Viper 6.5 DMR removed due to being repetitive/useless
- STS-25 removed (kept in code for now)
- Gladstone shotgun removed; it was just a 'worse' copy of the Regulator. Plus replaced with the Operator shotgun.

## Changelog
:cl:
add: Saiga added to Sergeant loadout.
add: Roe rifle added to trooper loadout options.
balance: Bounty changed to common-duty shotgun; replaced BS Grizzly.
balance: Redid stats for the Rushing Bull, moved it to be the CO's unique weapon.
fixes: Not really a 'fix' but changed the Rushing Bull's name to the Buffalo due to often confusion with the Bull shotgun.
remove: Buckler SMG; replaced with PPSH.
remove: Watchtower DMR; Vintorez replacement.
remove: Viper 6.5 DMR.
remove: Maxim and Pegasus removed off the BS teleporter/disk.
/:cl:

### CODE REVIEW

**Comment by ThePainkiller** [comment]
Welp. There goes my fucking on back and resprite of the BS version of the Maxim :)

File: code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm [hunk 1]
```diff
@@ -550,6 +538,20 @@
 		/datum/design/autolathe/ammo/rifle_lethal = 4,
 	)
 
+/obj/item/computer_hardware/hard_drive/portable/design/blackshield/strelki
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm:541]: if you can move these to blackshield_disks that would be lovely

File: code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm [hunk 2]
```diff
@@ -584,6 +586,21 @@
 		/datum/design/autolathe/ammo/sl_rifle,
 	)
 
+/obj/item/computer_hardware/hard_drive/portable/design/blackshield/zatvor
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm:589]: this one is the other one im refering to
